service_container/tags.rst

```diff
@@ -570,47 +570,113 @@ application handlers::
         }
     }
 
-.. tip::
+Prioritizing Tagged Services
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The tagged services can be prioritized using the ``priority`` attribute:
```

Comment:
To me a sentence is missing what I can achieve by using priority. I mean we are talking a lot about the how but not the why. A real world use case would be very helpful to get context. Thanks

Reply:
From my side it looks pretty obvious, but maybe I am not the best person to ask since I did not need this until now. I started looking into this on Symfony Hack Day. If you can share some use case, we can surely added it.

Reply:
What about expanding that sentence a little? Something like:
```rst
The tagged services can be prioritized using the ``priority`` attribute,
so a consumer can be injected a sorted collection:
```

Reply:
Updated. I tried to avoid using the word `consumer` since that can have different meanings and might confuse people. Thanks @HeahDude